python/googleapis/python-multi/Dockerfile

```diff
@@ -146,12 +144,12 @@ RUN for PYTHON_VERSION in 2.7.18 3.6.15 3.7.12 3.8.12 3.9.7 3.10.0; do \
   && rm -r "$GNUPGHOME"
 
 
-# Install pip on Python 3.6 only.
+# Install pip on Python 3.10 only.
```

Comment:
Can we use Python3.8 or 3.9 instead? 3.10 is not thoroughly tested or supported.

Reply:
Addressed in [8f71a30](https://github.com/googleapis/testing-infra-docker/pull/231/commits/8f71a30710254a529923d2503dd4c28f1af58bec)